en/components/grid-column-pinning.md

```diff
@@ -1,20 +1,24 @@
 ---
 title: Grid Column Pinning - Grid Feature
-_description: The Grid Column Pinning is a mechanism to pin the first few columns of a scrollable Grid. 
```

Comment:
I would recommend wording like: "The Grid Column Pinning is a mechanism to pin selected columns of a scrollable Grid that after pinning become first Grid columns".

Reply:
I will just use this part to keep it shorter because it is metadata.
"The Grid Column Pinning is a mechanism to pin selected columns of a scrollable Grid"